fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/FedoraTypesUtils.java

```diff
@@ -153,6 +168,12 @@
     });
 
     /**
+     * Check if a property is intentionally suppressed.
+     */
+    final static List<String> suppressed = Arrays.asList("jcr:mimeType", "jcr:lastModified");
```

Comment:
This also seems not to be used, and so I would be inclined to remove this along with the `Predicate` on the next line.